docs/how-to-guides/support-new-model.md

```diff
@@ -0,0 +1,688 @@
+# How to support a new model
+
+## How does Kernl optimize a model
+
+### Overview
+
+To optimize a model, Kernl uses [TorchDynamo](https://github.com/pytorch/torchdynamo) JIT compiler and provides a custom backend where we replace part of the [Torch FX](https://pytorch.org/docs/1.13/fx.html) graph with optimized kernels.
```

Comment:
FX is not yet defined, I would just say we replace a part of the model with our own kernels

Reply:
not very important.